src/scenarios/emqttb_scenario_pubsub_forward.erl

```diff
@@ -0,0 +1,195 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+-module(emqttb_scenario_pubsub_forward).
+
+-behavior(emqttb_scenario).
+
+%% First all subscribers connect and subscribe to the brokers, then
+%% the publishers start to connect and publish.  The default is to use
+%% full forwarding of messages between the nodes: that is, each
+%% publisher client publishes to a topic subscribed by a single
+%% client, and both clients reside on distinct nodes.
```

Comment:
This can be moved to `docs/src/conf.xml`, this way it will be rendered in the documentation.

Reply:
I've added a section there, but I think that having a short comment on the module helps having a quick understanding of the module, without having to render/view the XML.